packages/frontend/src/boot/common.ts

```diff
@@ -280,6 +280,19 @@ export async function common(createVue: () => App<Element>) {
 
 	removeSplash();
 
+	//#region Load default font
+	const def_arr = miLocalStorage.getItem('defaultFontFace')?.split('_');
+	const fontId = def_arr?.[0];
+	if (fontId && fontId !== 'system-ui') {
+		try {
+			await import(`@/styles-font/${fontId}.scss`);
```

Comment:
fixed it by renaming this font style

Reply:
If you look closely at this file, you will find that it is actually a collection of 6 different widely-used fonts(sharkey-default, system-ui, misskey-biz, roboto, arial, times). So I recommend to include it by default, since it will not query any external resources.

Reply:
https://github.com/Natsuki-Kaede/sharkey-stelpolva/blob/7c4f8364af060274e3ff54a3db2a4c5f1fd9a6a6/packages/frontend/src/styles-font/zhuanti.scss

same (`zhuanti.scss` contains both 'yishanbeizhuan' and 'chongxiseal')

Reply:
okay i will split them and fix the typo
the commits are a bit disorganized, I'll recreate another cleaner one 🥲